lib/adal-angular.js

```diff
@@ -175,16 +185,25 @@ if (typeof module !== 'undefined' && module.exports) {
                 var routeChangeHandler = function (e, nextRoute) {
                     if (nextRoute && nextRoute.$$route && isADLoginRequired(nextRoute.$$route, _adal.config)) {
                         if (!_oauthData.isAuthenticated) {
-                            _adal.info('Route change event for:' + $location.$$path);
+                            _adal.info('Route change event for:' + $location.$$url);
                             loginHandler();
                         }
                     }
                 };
 
-                var stateChangeHandler = function (e, nextRoute) {
-                    if (nextRoute && isADLoginRequired(nextRoute, _adal.config)) {
+                var stateChangeHandler = function (e, toState, toParams, fromState, fromParams){
```

Comment:
can you also share the ui-route config? my main concern is this will change how people use us today, and this is definitely not what we want.